### DISCUSSION TITLE
Using sample names for proteins in .faa file

### DISCUSSION THREAD

**Discussion by HilaryESR** [oschwengers/bakta#205] [Help wanted]
Is there a way to make bakta output proteins with the sample name as prefix, instead of what I presume is a locus tag?  For example currently outputs are like this:
>NPBBKN_00005 UDP-N-acetylglucosamine 1-carboxyvinyltransferase

But I want "NPBBKN" replaced with the name that I specify with the --prefix option.  

I need to combine bakta outputs from multiple samples into one file for AMR analysis and currently I have no way of knowing which proteins come from which sample.

**Comment by oschwengers** [comment]
Via the `--prefix` option one can set the output file name prefices. I think the best approach to achieve this, is to actually set the locus tag prefix via `--locus-tag`.  By this, you can set distinct locus tags for each sample and thus, proteins can be assigned to their sample.

- W/o `--compliant`: locus tag prefixes must contain between 1 and 24 alphanumeric characters or '_.-' signs.
- with `--compliant`: INSDC Locus tag prefixes must contain between 3 and 12 alphanumeric uppercase characters and start with a letter.

**Comment by HilaryESR** [comment]
Thanks, setting the locus tag works well
